tahrir/templates/error.html

```diff
@@ -0,0 +1,18 @@
+{% extends '_base.html' %}
```

Comment:
Note that this might regress on #370, though that issues was never explicitly closed (so I don't know that it was ever intentionally fixed) and weirdly, I can't find jQuery anywhere in the history of `404.html`, so I'm not sure where it was coming from before.